### ISSUE TITLE
[Bug] Is the guid persistence option available in v2.5.2 on Windows?

### ISSUE THREAD

**Issue by snltty** [xjasonlyu/tun2socks#385]
### Verify steps

- [X] Is this something you can **debug and fix**? Send a pull request! Bug fixes and documentation fixes are welcome.
- [X] I have searched on the [issue tracker](……/) for a related issue.

### Version

2.5.2

### Description

在windows，我使用这样的命令运行，编号会一直递增，然后注册表一直在增加记录
```
tun2socks-windows-amd64.exe -device tun://linker?guid={335B2CEC-F95A-46B8-8791-7E6134A5DE33} -proxy socks5://127.0.0.1:5555
```

在注册表 ，编号递增会增加新记录
```
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\NetworkList\Signatures\Unmanaged

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\NetworkList\Profiles
```

**Comment by xjasonlyu** [comment]
No, the guid persistence option is currently not available in v2.5.2. But you can always build tun2socks from the latest branch by yourself.

**Comment by snltty** [comment]
OK, thanks
